pkg/perf/monitor.go

```diff
@@ -86,3 +94,13 @@ func (pm *PerformanceMonitor) Run(ctx context.Context) error {
 	pm.scheduler.StartAsync()
 	return nil
 }
+
+type zbusClient struct{}
+
+func withZbusClient(ctx context.Context, client zbus.Client) context.Context {
+	return context.WithValue(ctx, zbusClient{}, client)
+}
+
+func getZbusClient(ctx context.Context) zbus.Client {
```

Comment:
```suggestion
func GetZbusClient(ctx context.Context) zbus.Client {
```

this should be public because external tasks can still use it